### PR TITLE
Instruction

### PR DESCRIPTION
**Pull request by bojum**
This is setup the team would like to have and want to run it by you before publishing it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,39 @@
...
+      \<blank line\>
+      
+      \# \<Your project name\>
+
+Note that there is a whitespace between ‘#’ and a project name. Otherwise, the file will not be captured by the class index file.
+
+2. The second line of the text that part 1. asks you to add **must** be the only line with a single ‘#’. If there are parts in the document that need ‘#’s to bold the text, use more than 1 ‘#’s. 
+
+3. When the git repo is initiated, there will be a README file that lives outside the project folder you have created. **DO NOT ADD THIS README FILE when you commit.** It will create a conflict and we will not be able to merge your work into the class repo. 
+
```
Comment (jtr13) [inline README.md:33]: To be more clear here, say: Do not make any changes to the README outside of your folder. If you do accidentally, do not commit those changes.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,39 @@
 Community contributions for STAT GR 5702 Fall 2019
 
 This is an amazing class.
```
Comment (jtr13) [inline README.md:4]: You can delete the "amazing class" line. 

Add some explanation about bookdown and a link to an example bookdown web site.

**Comment by jtr13** [comment]
This looks good.  One question I have is how you will handle the order of chapters. You may want to create parts, see here for example: https://github.com/jtr13/EDAV/blob/master/_bookdown.yml

It's up to you, but you may want to consider asking students to come up with a descriptive name for their folder rather than using their UNIs. Assuming you will be manually adding the chapter names to the _bookdown.yml, it may get confusing to remember which project is which, especially since the UNIs are often quite similar and hard to remember. 

Another idea I had is for me to recruit students to serve as testers... would that be helpful? It's good to test the system with users who don't have write access.

**Comment by bojum** [comment]
We were thinking of having a .yaml file that ordered all the .rmd file in the directory. @Yingyu-Cao can confirm/provide more info about this. 

On second thought, I agree with you on using descriptive names. Would it matter what format students use? I personally prefer no-whitespace names.

**Comment by jtr13** [comment]
Agree, no whitespaces. I'm going to merge the pull request as is, however you can keep editing it directly.

**Pull request merged**